qdrant/v1.1.x/comparisons.md

```diff
@@ -0,0 +1,69 @@
+| **Query**      | **Search**     | Name of the method used to find the nearest neighbors for a given vector, possibly with some additional filters applied on top.                                                                                                                                                |
+| N/A            | **Scroll**     | Pinecone does not offer a way to iterate through all the vectors in a particular index. Qdrant has a `scroll` method to get them all without using search.                                                                                                                     |
+
+#### Known metadata limitations
+
+1. Pinecone does not support any JSON metadata, but a flat structure with strings, numbers, booleans, or lists of strings used as values. Qdrant accepts any JSON object as a payload, even nested structures.
+2. NULL values are not supported in Pinecone metadata but are handled properly by Qdrant.
+3. The maximum size of Pinecone metadata is 40kb per vector. 
+4. Pinecone, unlike Qdrant, does not support geolocation and filtering based on geographical criteria.
+
+It is worth mentioning, that **Pinecone will automatically create metadata indexes for all the fields**. Qdrant assumes you know
+your data and your future queries best, so it's up to you to choose the fields to be indexed. Thus, **you need to explicitly 
+define the payload indexes while using Qdrant**.
+
+### Supported technologies
+
+Both tools support various programming languages providing official SDKs.
+
+|                           | Pinecone             | Qdrant               |
+|---------------------------|----------------------|----------------------|
+| **Python**                | ✅                    | ✅                    |
+| **JavaScript/TypeScript** | ✅                    | ✅                    |
+| **Rust**                  | ❌                    | ✅                    |
+| **Go**                    | ❌                    | ✅                    |
+
+There are also various community-driven projects aimed to provide the support for the other languages, but those are not officially 
+maintained, thus not mentioned here. However, it is still possible to interact with both engines through the HTTP REST or gRPC API. 
+That makes it easy to integrate with any technology of your choice.
+
+If you are a Python user, then both tools are well-integrated with the most popular libraries like LangChain, LlamaIndex, Haystack, and more. 
+Using any of those libraries makes it easier to experiment with different vector databases, as the transition should be seamless.
+
+### Comparison to Qdrant Cloud
+
+Both Pinecone and Qdrant Cloud offer a free cloud tier to check out the services, even without providing the credit card details.
+Each free tier should be enough to keep around 1M of 768-dimensional vectors, but it may vary depending on the additional attributes
+stored with vectors. 
```

Comment:
Maybe that we can do many collections per cluster? So that you can build multiple different small demos on a free tier, not just one as on Pinecone

Reply:
It is mentioned in the table already, but that's a good idea to emphasize that again. Thanks!